cms/grading/__init__.py

```diff
@@ -448,7 +456,7 @@ def compilation_step(sandbox, commands):
 
 
 def evaluation_step(sandbox, commands,
-                    time_limit=0.0, memory_limit=0,
+                    time_limit=0.0, memory_limit=0, max_processes=1,
```

Comment:
Can you explain why? evaluation_step is used by more than just Batch.py (the only thing which uses LANGUAGE_TO_MAXPROCESSES) so it should have a sane default for all things, right? evaluation_step shouldn't need to care about the language so it doesn't really make sense to the default from LANGUAGE_TO_MAX_PROCESSES.
Otherwise it could default to None (no limit) like time/memory but that also seems a bit odd